src/character.cpp

```diff
@@ -3323,8 +3323,28 @@ units::mass Character::bodyweight() const
 
 int Character::height() const
 {
+    int height = init_height;
+    int height_pos = 15;
+
+    const std::vector<int> v = { 290, 240, 190, 140, 90 };
```

Comment:
Should be `static` and probably an `std::array<int, 5>` as you're never changing it.